library/lua/gui/widgets/text_area.lua

```diff
@@ -0,0 +1,166 @@
+-- Multiline text area control
+
+local Panel = require('gui.widgets.containers.panel')
+local Scrollbar = require('gui.widgets.scrollbar')
+local TextAreaContent = require('gui.widgets.text_area.text_area_content')
+
+TextArea = defclass(TextArea, Panel)
+
+TextArea.ATTRS{
+    init_text = '',
```

Comment:
Do you think it makes sense to switch this to `text` instead of `init_text`? It feels like `text` might align more closely with the API of other widgets like `EditField`, `Label`

Reply:
I am afraid it will consfuse users and they will treat it as writeable property - where it is not. they need to to use `:setText(...)`